vnext/Microsoft.ReactNative.IntegrationTests/JsiSimpleTurboModuleTests.cpp

```diff
@@ -0,0 +1,124 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+// See that we can safely reload react instance when we use C++ TurboModules.
+
+#include "pch.h"
+#include <ReactCommon/TurboModule.h> // This comes from the react-native package.
+#include <ReactCommon/TurboModuleUtils.h> // This must come from the react-native package, but we use a local version to fix issues.
```

Comment:
i don't see a fork of this file, is the comment out of date?

Reply:
Yes, it was a copy and paste comment from the previous test. In the meantime the `TurboModuleUtils.h` was fixed in the react-native repo and the comment became out-of-date. I have removed the comments in both files: https://github.com/microsoft/react-native-windows/pull/7356/commits/7547d7b2890b2afcd184d724566ee8238a8b15ba